### PR TITLE
install/bin/stdin-killer: macOs (Darwin) compatibility

### PR DESCRIPTION
**Pull request by leonid-s-usov**

### CODE REVIEW

**Comment by leonid-s-usov** [comment]
The failing check is addressed by https://github.com/ceph/teuthology/pull/1894

File: teuthology/task/install/bin/stdin-killer [hunk 1]
```diff
@@ -42,7 +42,7 @@ NAME = "stdin-killer"
 log = logging.getLogger(NAME)
 PAGE_SIZE = 4096
 
-POLL_HANGUP = select.POLLHUP | select.POLLRDHUP | select.POLLERR
+POLL_HANGUP = select.POLLHUP | select.POLLRDHUP if hasattr(select, 'POLLRDHUP') else 0 | select.POLLERR
```
Comment (batrick) [inline teuthology/task/install/bin/stdin-killer:45]:
```suggestion
POLL_HANGUP = select.POLLHUP | (select.POLLRDHUP if hasattr(select, 'POLLRDHUP') else 0) | select.POLLERR
```

**Review by batrick — CHANGES_REQUESTED** [review]
otherwise LGTM

File: teuthology/task/install/bin/stdin-killer (hunk 1 above)
Comment (leonid-s-usov) [reply teuthology/task/install/bin/stdin-killer]: Done

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
> @batrick hm, I do not have access to teuthology on readthedocs, do you mind adding me?

I'm not sure. You may be better asking @neha-ojha

**Comment by zmc** [comment]
@batrick you're listed as an admin, but she's not: https://readthedocs.org/projects/teuthology/

**Comment by zmc** [comment]
> > @batrick you're listed as an admin, but she's not: https://readthedocs.org/projects/teuthology/
> 
> Do you have a readthedocs username? Your emails didn't work in their lookup system. Mine for example is:
> 
> https://readthedocs.org/profiles/batrick/
> 
> which I login to with via github auth.

Yep, [same for me](https://readthedocs.org/profiles/zmc/) - I hadn't added my RH email to the profile, but I've done that just now.

**Comment by batrick** [comment]
> > > @batrick you're listed as an admin, but she's not: https://readthedocs.org/projects/teuthology/
> > 
> > 
> > Do you have a readthedocs username? Your emails didn't work in their lookup system. Mine for example is:
> > https://readthedocs.org/profiles/batrick/
> > which I login to with via github auth.
> 
> Yep, [same for me](https://readthedocs.org/profiles/zmc/) - I hadn't added my RH email to the profile, but I've done that just now.

I've invited you to both projects.

**Comment by zmc** [comment]
> > > > @batrick you're listed as an admin, but she's not: https://readthedocs.org/projects/teuthology/
> > > 
> > > 
> > > Do you have a readthedocs username? Your emails didn't work in their lookup system. Mine for example is:
> > > https://readthedocs.org/profiles/batrick/
> > > which I login to with via github auth.
> > 
> > 
> > Yep, [same for me](https://readthedocs.org/profiles/zmc/) - I hadn't added my RH email to the profile, but I've done that just now.
> 
> I've invited you to both projects.

Thanks! Appreciate it.

**Pull request merged**